aspnetcore/blazor/hybrid/tutorials/wpf.md

```diff
@@ -64,6 +64,19 @@ At the top of the project file, change the SDK to `Microsoft.NET.Sdk.Razor`:
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 ```
 
+:::moniker range=">= aspnetcore-10.0"
+
+In the project file's existing `<TargetFramework>` property, add a Windows 10 version number:
+
+```xml
+<TargetFramework>net10.0-windows10.0.17763.0</TargetFramework>
```

Comment:
Passing note/concern ...

We're saying "existing `<TargetFramework>` property," but the preceding is only showing one TFM. Should we show a placeholder for existing TFMs?

```suggestion
<TargetFramework>{EXISTING TARGET FRAMEWORKS};net10.0-windows10.0.17763.0</TargetFramework>
```